app/models/waste_carriers_engine/finance_details.rb

```diff
@@ -26,6 +26,15 @@ def self.new_renewal_finance_details(transient_registration, method, current_use
       finance_details
     end
 
+    # This amounts are used in a number of finance operations.
+    def amount_in_excess
+      @_amount_in_excess ||= balance <= 0 ? (balance * -1) : 0
+    end
+
+    def amount_to_pay
```

Comment:
Follows on from my previous comment

```suggestion
    def unpaid_balance
```